tests/acceptance/features/lib/BruteForceProtectionPage.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * ownCloud
+ *
+ * @author Artur Neumann <artur@jankaritech.com>
+ * @copyright Copyright (c) 2018 Artur Neumann artur@jankaritech.com
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Page;
+
+use Behat\Mink\Session;
+use SensioLabs\Behat\PageObjectExtension\PageObject\Exception\ElementNotFoundException;
+
+/**
+ * PageObject for the brute force protection Page
+ *
+ */
+class BruteForceProtectionPage extends OwncloudPage {
+
+	/**
+	 *
+	 * @var string $path
+	 */
+	protected $path = '/index.php/settings/admin?sectionid=security';
+	private $tresholdTimeInputId = 'bfp-threshold-time';
```

Comment:
s/tresholdTimeInputId/thresholdTimeInputId

global search for "treshold" and fix typo.